test/Extension.tests/UnitTests.cs

```diff
@@ -43,6 +44,36 @@ private void DummyMethod(JObject e)
             Assert.Equal($"Unable to bind {testObject} to type {arrayParam[0].ParameterType}", invalidException.Message);
         }
 
+        private void DummyMethodForOutputBinding(EventGridEvent message) { }
+
+        [Fact]
+        public async Task OutputBindingAsyncTest()
```

Comment:
this is the test for `BindAsync()` method...which is called in triggerbinding pipeline, should not apply to you